mono/metadata/w32error-unix.c

```diff
@@ -67,8 +67,9 @@ mono_w32error_unix_to_win32 (guint32 error)
 	case EPIPE: return ERROR_WRITE_FAULT;
 	case ELOOP: return ERROR_CANT_RESOLVE_FILENAME;
 #ifdef ENODEV
-	case ENODEV: return ERROR_DEV_NOT_EXIST;
+	case ENODEV:
 #endif
+	case ENXIO: return ERROR_DEV_NOT_EXIST;
```

Comment:
Are we sure `ENXIO` is defined on all platforms?

Reply:
no, updated the PR. thanks!